src/data/data.tsx

```diff
@@ -0,0 +1,129 @@
+import { MockEvent } from "../types/MockEvents";
+
+export const mockEvents: MockEvent[] = [
+  {
```

Comment:
There is some data on Figma to use for this. 

![image](https://github.com/nology-tech/Union-Client-Project/assets/55165280/bf70f88a-264c-414e-8a8f-ce0cae3ba4b9)